src/styles/header.scss

```
@@ -0,0 +1,64 @@
+.header {
+  background-color: $white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  @media (max-width: 768px) {
+    padding: 0 20px;
+    font-size: 10px;
+  }
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+    flex-shrink: 0;
+
+  }
+
+  &-nav__list {
```

should be
```suggestion
  &__nav-list {
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**